### PR TITLE
Add linux aarch64 build

### PR DESCRIPTION
**Pull request by tanakadesilva**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

* Add linux_aarch64 build
* Fix the osx arm64 tarball conditional

### CODE REVIEW

**Comment by tanakadesilva** [comment]
@conda-forge-admin, please rerender

**Comment by tanakadesilva** [comment]
@conda-forge-admin, please rerender

**Comment by tanakadesilva** [comment]
@conda-forge-admin, please rerender

**Comment by tanakadesilva** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,15 +5,17 @@ package:
   version: {{ version }}
 
 source:
-  url: https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-{{ version }}-linux-x86_64.tar.gz  # [linux]
-  sha256: b24ba27d57463e08db859b7941cf4d7c33cb3af5865095639e3e0b2055cbec66  # [linux]
-  url: https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-{{ version }}-darwin-x86_64.tar.gz  # [osx and build_platform == "osx-64"]
-  sha256: 85d71c956529453f246e3b41339903b8992aa326f15ea6bc36f2ec79e76c7d02  # [osx and build_platform == "osx-64"]
-  url: https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-{{ version }}-darwin-arm.tar.gz  # [osx and build_platform == "osx-arm64"]
```
Comment (tanakadesilva) [inline recipe/meta.yaml:12]: Shouldn't this be dependent on the target platform and not the build? given that the`osx_arm64` target is built `osx_64`.

Comment (tanakadesilva) [inline recipe/meta.yaml:12]: Shouldn't this be dependent on the target platform and not the build? given that the `osx_arm64` target is built `osx_64`.

**Comment by ptonner** [comment]
I'm interested in having the google-cloud-sdk feedstock support linux aarch64. Is it possible to merge/update this PR? happy to make the changes if needed, but would need to be pointed in the right direction as I've not contributed to conda forge before

**Pull request closed without merging**